stack/templates/deployment.yaml

```diff
@@ -71,9 +71,15 @@ spec:
           {{- end }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
-          {{- with .Values.volumeMounts }}
+          {{- if or .Values.persistence.enabled .Values.volumeMounts}}
```

Comment:
Can you add defaults to `values.yaml` for the `persistence` fields? That's new, right?

Reply:
Yes, I added them here: https://github.com/chanzuckerberg/argo-helm-charts/blob/GPU-271/pvc-creation-support/stack/values.yaml#L172